scripts/superdesk-search/services/TagService.js

```diff
@@ -136,6 +137,23 @@ export function TagService($location, desks, userList, metadata, search, ingestS
         });
     }
 
+    function initRemovedFacets(params) {
+        _.each(EXCLUDE_AGGREGATES, function(label, key) {
+            tags.removedFacets[key] = [];
+            if(angular.isDefined(params[key])) {
```

Comment:
This is also just an optional style change, but you could reduce indentation by doing:
```js
if (!angular.isDefined(params[key])) {
    return;
}
```
and then continue the logic without increasing indentation because of the `if`.

Either way, you've missed a space between `if` and the `(` which is odd that it wasn't picked up by the linters.